server/app/lib/actions/fusor/deployment/open_shift/install_ose.rb

```diff
@@ -28,7 +28,7 @@ def run
             ::Fusor.log.info "================ OpenShift InstallOSE run method ===================="
             deployment = ::Fusor::Deployment.find(input[:deployment_id])
             opts = parse_deployment(deployment)
-            launcher = OSEInstaller::Launch.new("#{Rails.root}/tmp/#{deployment.label}", ::Fusor.log)
+            launcher = OSEInstaller::Launch.new("#{Rails.root}/tmp/#{deployment.name}", ::Fusor.log_file_dir(deployment.label, deployment.id), ::Fusor.log)
```

Comment:
Is deployment.name desired here, are they any issues if the name contains strange characters?

I